cs481-ekh/f21-welolo
# Closes #49 (#83)

Verification Steps:
1. Run './build.sh' from the root project folder
2. In two separate terminals run 'npm start' in both the client and server folder
3. In a third terminal, run 'npx cypress open' from the Welolo folder
4. Click on sms_payment.js to run the test
5. Notice the test fails due to a bug in the application
